tests/e2e/tests/content-manager/history.spec.ts

```diff
@@ -252,7 +252,9 @@ describeOnCondition(edition === 'EE')('History', () => {
       await clickAndWait(page, page.getByRole('link', { name: /Create new entry/, exact: true }));
       await page.waitForURL(ARTICLE_CREATE_URL);
       await page.getByRole('textbox', { name: 'title' }).fill('Being from Kansas');
-      await page.getByRole('textbox', { name: 'slug' }).fill('being-from-kansas');
+      await page
+        .getByRole('textbox', { name: 'slug This value is unique for the selected locale' })
+        .fill('being-from-kansas');
```

Comment:
I'm all for it if the tests pass reliably, but I admit I don't understand it. Do you know what the error in the CI currently is? Does it match several items? because it should be able to find a match even if we don't provide the whole dom text

Reply:
The test wasn't even failing here. It's failing later when trying to find a value in the listview gridcell, but it can't because it was never filled. But I took another look and it does look like this is always filled even though sometimes on webkit there is no value in the listview.

> because it should be able to find a match even if we don't provide the whole dom text

Yeah I had the same thought but since it's only failing for webkit I was assuming it was a limitation of that browser 🤷‍♂️ 

This does seem to fix this issue consistently (I've run the same fix on the review workflow pr as well) but I can't really explain why. 

In other news it looks like there are other issues with the history tests. So looks like I'll be digging deeper.